### PR TITLE
Add Eligibility screens

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

* Add three new screens for entering eligibility information

## Screenshots of UI changes
localhost_3000_applications_57685284-489a-47c4-a521-18d64d918012_tasks_eligibility_pages_eligibility-reason
localhost_3000_applications_57685284-489a-47c4-a521-18d64d918012_tasks_eligibility_pages_release-date
localhost_3000_applications_57685284-489a-47c4-a521-18d64d918012_tasks_eligibility_pages_accommodation-required-from-date
localhost_3000_applications_57685284-489a-47c4-a521-18d64d918012_tasks_check-your-answers_pages_review

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Add Eligibility screens](https://trello.com/c/aVOIbvCV/1099-add-eligibility-screens)

File: server/form-pages/apply/accommodation-need/eligibility/eligibilityReason.test.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+      expect(page.body).toEqual(body)
+    })
+  })
+
+  itShouldHavePreviousValue(new EligibilityReason({}, application), 'dashboard')
+  itShouldHaveNextValue(new EligibilityReason({}, application), 'release-date')
+
+  describe('errors', () => {
+    it('returns an empty object if the reasone is populated', () => {
```
Comment (tahb) [inline server/form-pages/apply/accommodation-need/eligibility/eligibilityReason.test.ts]: "reasone"

File: server/views/applications/pages/eligibility/eligibility-reason.njk [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{% extends "../layout.njk" %}
+
+{% block questions %}
+
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-l">{{ section.title }}</span>
+    {{ page.title }}
+  </h1>
+
+	{{ formPageRadios({
```
Comment (tahb) [inline server/views/applications/pages/eligibility/eligibility-reason.njk]: Indentation may have gone awry here

**Review by tahb — APPROVED** [review]

File: server/views/applications/pages/eligibility/eligibility-reason.njk (hunk 2 above)
Comment (bryangaledxw) [reply server/views/applications/pages/eligibility/eligibility-reason.njk]: Thanks, a tab crept in!

**Pull request merged**